### PR TITLE
Update records-management.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to point 3 in the footnote per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10240

The records can be moved to other document libraries within the same site

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10240

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
fixes # https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10240

File: microsoft-365/compliance/records-management.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Deleting labeled items in SharePoint and OneDrive can be blocked as a tenant set
 When you apply a retention label to a list item that has a document attachment, that document doesn't inherit the retention settings and can be deleted from the list item. In comparison, if that list item was declared a record with a retention label, the document attachment would inherit the retention settings and couldn't be deleted.
 
 <sup>3</sup>
-Containers include SharePoint document libraries, OneDrive accounts, and Exchange mailboxes.
+Containers include SharePoint Site, OneDrive accounts, and Exchange mailboxes.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/records-management.md]:
```suggestion
Containers include SharePoint sites, OneDrive accounts, and Exchange mailboxes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cabailey Please merge this PR. Thanks

**Comment by cabailey** [comment]
Confirmed by engineering - thanks for flagging, @kstat

**Comment by cabailey** [comment]
#sign-off, merging

**Pull request merged**